MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -323,6 +345,28 @@ final class MockCGMManagerSettingsViewController: UITableViewController {
                 vc.measurementFrequencyDelegate = self
                 show(vc, sender: sender)
             }
+        case .glucoseThresholds:
+            let vc = GlucoseEntryTableViewController(glucoseUnit: glucoseUnit)
+            vc.indexPath = indexPath
+            vc.glucoseEntryDelegate = self
+            switch GlucoseThresholds(rawValue: indexPath.row)! {
+            case .cgmLowerLimit:
+                vc.title = "CGM Lower Limit"
+                vc.contextHelp = "The glucose value that marks the lower limit of the CGM. Any value at or below this value is presented at `LOW`. This value must be lower than the urgent low threshold otherwise the app will crash."
```

Comment:
Will the app actually crash?

Reply:
yes. The switch that uses these values to determine the type of the glucose value cannot create the ranges.

Reply:
However, I'll update to just adjust the entered value to something that won't crash, since this note isn't obvious and the crash would be confusing. a change to value is also confusing, but hopefully when that happens they will read the warning.